app/presenters/prior_authority/check_answers/base.rb

```diff
@@ -49,7 +49,6 @@ def row_content(head_key, text, actions = [], head_opts = {})
         {
           key: {
             text: heading,
-            classes: 'govuk-summary-list__value-width-50'
```

Comment:
without this I thought each section would be a different width based on the size of the text leading to a non-uniform appearance???

Reply:
No, the standard width for the key is 30%